lib/zold/commands/pay.rb

```diff
@@ -103,7 +103,16 @@ def pay(from, invoice, amount, details, opts)
       @log.debug("#{amount} sent from #{from} to #{txn.bnf}: #{details}")
       @log.debug("Don't forget to do 'zold push #{from}'")
       @log.info(txn.id)
+      notify_of_tax_debt(from)
       txn
     end
+
+    # @todo #79:40min Add tests, extract to a module(?).
```

Comment:
@ledestin run [pdd](https://github.com/yegor256/pdd#how-to-format) to correctly format this puzzle.